packages/frontend/src/pages/server-information/LocalServer.svelte

```diff
@@ -0,0 +1,89 @@
+<script lang="ts">
+import { NavPage, Input, ErrorMessage } from '@podman-desktop/ui-svelte';
+import { studioClient } from '/@/utils/client';
+import { configuration } from '/@/stores/extensionConfiguration';
+import { onMount, onDestroy } from 'svelte';
+
+// The AI Lab Port is the bind value to form input
+let aiLabPort: number | undefined = $state(undefined);
+let invalidPort: boolean = $state(false);
+let errorMsg: string | undefined = $state(undefined);
+
+onMount(() => {
+  const port = $configuration?.apiPort;
```

Comment:
if there are some external changes to the port, it won't be reflected in the UI as it's only reading apiPort value when displaying the first time the value

so let say, another process update the apiPort configuration value, this page will still display the same value than when the page was loaded

Reply:
Yes, I think that this is a reason why you need to restart the extension too, to reflect the changes